lib/reporters/base.js

```diff
@@ -27,6 +27,11 @@ exports = module.exports = Base;
 
 var isatty = tty.isatty(1) && tty.isatty(2);
 
+/**
+ * Save log references to avoid tests interfering (see GH-3604).
+ */
+var println = console.log;
```

Comment:
how about `log`

Reply:
also, I'm not sure if `console.log` needs a context in all environments we support.  in other words, if `var log = console.log; log('foo')` works, we're ok.  

I know it's fine to do this in Node.js and chromium, but not sure about IE11 or other browsers.  `console` is completely nonstandard afaik, so we have to be careful.

Reply:
if we *do* discover this is not supported in something, then the fix would be

```js
var log = console.log.bind(console);
```

Reply:
I branched directly from upstream so that the PR would run the full Karma features. I also tested it locally on IE11 and it seemed to work as expected. So I think its ok.

Reply:
Why not `consoleLog` rather than `log` (which is used for _lots_ of things)?